spec/mongo/auth/scram256/negotiation_spec.rb

```diff
@@ -0,0 +1,562 @@
+require 'spec_helper'
+require 'cgi'
+
+module Mongo
+  class Server
```

Comment:
This is monkey-patched to facilitate testing that SCRAM-SHA-256 is used when both SCRAM-SHA-1 and SCRAM-SHA-256 are available for a given user.

Reply:
This one